int32 user_answer
---
float32 user_coeff
